noether_filtering/include/noether_filtering/cloud/mls_smoothing_filter.h

```diff
@@ -0,0 +1,44 @@
+#ifndef NOETHER_FILTERING_CLOUD_MLS_SMOOTHING_FILTER_H
+#define NOETHER_FILTERING_CLOUD_MLS_SMOOTHING_FILTER_H
+
+#include "noether_filtering/filter_base.h"
+
+namespace noether_filtering
+{
+namespace cloud
+{
+template<typename PointT>
+class MLSSmoothingFilter : public FilterBase<pcl::PointCloud<PointT>>
+{
+public:
+  using T = typename pcl::PointCloud<PointT>;
+
+  using FilterBase<T>::FilterBase;
```

Comment:
What does this do?

Reply:
This allows me to use the same constructors as defined in the base class. If I didn't do this I would have to define my own constructor(s) to mirror those in the base class, which in this case is unnecessary